### PR TITLE
Modernize imports of rose.server

### PR DESCRIPTION
**Pull request by moshe742**
for issue #192

### CODE REVIEW

File: rose/server/game.py [hunk 1]
```diff
@@ -3,9 +3,9 @@
 from twisted.internet import reactor, task
 
 from rose.common import actions, config, error, message, obstacles
-import track
-import player
-import score
+from . import track
+from . import player
+from . import score
```
Comment (nirs) [inline rose/server/game.py]: Looks good

File: rose/server/player_test.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import player
+from . import player
```
Comment (nirs) [inline rose/server/player_test.py]: Looks good

File: rose/server/score_test.py [hunk 3]
```diff
@@ -1,8 +1,8 @@
 from rose.common import actions, config, obstacles
-import track
-import player
-import score
-import pytest
+from . import track
+from . import player
+from . import score
```
Comment (nirs) [inline rose/server/score_test.py]: Looks good...

File: rose/server/score_test.py [hunk 4]
```diff
@@ -1,8 +1,8 @@
...
+from . import pytest
```
Comment (nirs) [inline rose/server/score_test.py]: But pytest is not a module this package, but a 3rd party module. This change is not needed.

**Comment by nirs** [comment]
@moshe742, the tests fail with this change, see:
https://travis-ci.org/RedHat-Israel/ROSE/builds/290612341

File: rose/server/score_test.py (hunk 4 above)
Comment (moshe742) [reply rose/server/score_test.py]: First, how can I run the tests on my machine?
I don't have for some reason pytest on the virtualenv (or should I run some other command?)

I fixed the import and did another commit and push but not seeing it here, should I do another pull request?

Comment (nirs) [reply rose/server/score_test.py]: To install the development packages:

    pipenv install --dev

(We need to document this in the readme)

If you fix this in the next commit, pushing again to github should show it here.

If you amended the original commit, you need to push with --force to replace the original commit on gituhub,

Your PR will be updated automatically in both ways.

**Comment by nirs** [comment]
@moshe742 looks like you have bad rebase in your history, the first patch appears twice.

You can fix this by interactive rebase:

```
git checkout imports_server
git rebase -i master
```

Now delete the patch appearing twice, save and quit.

**Comment by nirs** [comment]
@moshe742 can you rebase?

**Comment by nirs** [comment]
@moshe742, I rebased and merge your changes in #1bf6c1bfe63ceb except the first commit which introduces pep8 violation. Thanks!

**Pull request closed without merging**